src/main/java/com/jm/online_store/service/impl/UserServiceImpl.java

```diff
@@ -165,4 +174,51 @@ public boolean activateUser(String token, HttpServletRequest request) {
                 .map(Optional::get)
                 .collect(Collectors.toSet());
     }
+
+    @Override
+    @Transactional
+    public void updateUserImage(Long userId, MultipartFile file) {
+        String originalFilename = StringUtils.cleanPath(file.getOriginalFilename());
+        if (!file.isEmpty()) {
+            deleteUserImage(userId);
+            Path fileNameAndPath = Paths.get(uploadDirectory, originalFilename);
+            try {
+                byte[] bytes = file.getBytes();
+                Files.write(fileNameAndPath, bytes);
+                //Set user's profile picture
+                User user = userRepository.findById(userId).get();
+                user.setProfilePicture(originalFilename);
+                userRepository.save(user);
+            } catch (IOException e) {
+                log.debug("Failed to store file {} {}", fileNameAndPath, e.getMessage());
```

Comment:
log.debug("Failed to store file: {}, because: {}", fileNameAndPath, e.getMessage());